app/connectors/FileUploadConnector.scala

```diff
@@ -26,29 +26,22 @@ import scala.concurrent.Future
 
 trait FileUploadConnector extends ServicesConfig {
 
-  val http: HttpPost = WSHttp
-
+  val http: HttpPost
   lazy val serviceUrl = baseUrl("file-upload")
   lazy val serviceUrlSuffix = getString("file-upload-url-suffix")
 
-  def getEnvelope()(implicit hc: HeaderCarrier): Future[Option[String]] = {
-    val fileUploadUri = s"$serviceUrl/$serviceUrlSuffix"
-    val requestBody = Json.parse("""{"callbackUrl": "ourCallbackUrl"}""".stripMargin)
-    http.POST[JsValue, Option[String]](fileUploadUri, requestBody,Seq())(implicitly, rds = responseHandler, hc, MdcLoggingExecutionContext.fromLoggingDetails(hc))
-  }
+  def getEnvelope()(implicit hc: HeaderCarrier): Future[HttpResponse] = {
```

Comment:
@RaminEsfandiari please change the of the method createEnvelope as we are doing a post of the envelope to file-uplaoad